jdaviz/configs/cubeviz/plugins/parsers.py

```diff
@@ -489,15 +473,13 @@ def _parse_ndarray(app, file_obj, data_label=None, data_type=None,
 
     if data_type == 'flux':
         app.add_data_to_viewer(flux_viewer_reference_name, data_label)
-        app.add_data_to_viewer(spectrum_viewer_reference_name, data_label)
         app._jdaviz_helper._loaded_flux_cube = app.data_collection[data_label]
     elif data_type == 'uncert':
         app.add_data_to_viewer(uncert_viewer_reference_name, data_label)
         app._jdaviz_helper._loaded_uncert_cube = app.data_collection[data_label]
 
 
-def _parse_gif(app, file_obj, data_label=None, flux_viewer_reference_name=None,
-               spectrum_viewer_reference_name=None):  # pragma: no cover
+def _parse_gif(app, file_obj, data_label=None, flux_viewer_reference_name=None):  # pragma: no cover
```

Comment:
Baby Shark still would play but only via the buttons in Slice plugin. Spectrum viewer slider is broken. 😿

Reply:
I tried to make it so unitless data would still pass through the extraction, but there were some tests that still don't make sense to work through there.  I still think we can/should consider a more general cube extraction plugin for the non-spectral case (as we work to generalize the code).  If you have any suggestions to make it work here, I'm open to them, but also don't want this to blow up with the addition of another plugin now.

Reply:
Raw numpy array was okay, so I won't worry too much about it. I was just making a note for future reference.

I thought about inserting a dummy spectrum for this case just so the slider would work, but it is too hacky.